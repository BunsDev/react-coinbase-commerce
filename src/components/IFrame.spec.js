import React from 'react';
import IFrame from './IFrame';

jest.mock('../utils');

test('it renders and matches its snapshot', () => {
  const wrapper = shallow(
    <IFrame checkoutId={'aaaa'}/>
  );
  expect(wrapper).toMatchSnapshot();
});
