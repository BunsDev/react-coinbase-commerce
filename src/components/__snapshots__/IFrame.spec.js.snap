// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it renders and matches its snapshot 1`] = `
<IFrame
  checkoutId="aaaa"
>
  <div
    className="coinbase-commerce-iframe-container"
  >
    <div
      className="commerce-loading-spinner"
    />
    <iframe
      allowtransparency="yes"
      className="coinbase-commerce-iframe"
      frameBorder="no"
      scrolling="no"
      src="https://commerce.coinbase.com/embed/checkout/aaaa?origin=test://test.com&version=1&buttonId=undefined"
    />
  </div>
</IFrame>
`;
